cucumber/authenticators/features/support/authenticator_helpers.rb

```diff
@@ -53,6 +53,14 @@ def forbidden?
     http_status == 403
   end
 
+  def bad_gateway?
+    http_status == 502
+  end
+
+  def read_timeout?
```

Comment:
i would change this method name. the behaviour of the rest of these method i to check the http status but here it is different and misleading.